src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java

```diff
@@ -46,6 +46,17 @@ public void onEnable() {
         getServer().getPluginManager().registerEvents(new Events(), this);
 
         // Category
```

Comment:
Move comment below enchant crap and also comment on enchant crap

Reply:
alright, also do u guys care for javadocs